nntrainer/src/tensor.cpp

```diff
@@ -239,6 +239,20 @@ Tensor Tensor::add(float const &value) {
   return result;
 }
 
+void Tensor::saxpy(const unsigned int N, const float alpha, const float *X,
+                   const int incX, float *Y, const int incY) {
```

Comment:
I think we have to make this dummy func in different file for better managing. Also we have to add more blas func like gemm.

Reply:
yes.